apps/wing-console/console/server/src/utils/test-runner/simulator-manager.ts

```diff
@@ -0,0 +1,58 @@
+import type { Simulator } from "../../wingsdk.js";
+import { createCompiler, type Compiler } from "../compiler.js";
+import { createSimulator } from "../simulator.js";
+
+export const createSimulatorManager = ({
+  wingfile,
+  platform,
+  watchGlobs,
+}: {
```

Comment:
Can you explain what's the purpose of this manager?

Reply:
The comments don't explain the behavior. From what I can tell:

- `createSimulatorManager ` will create one compiler (watcher) for the given wingfile when `useSimulatorInstance` is called for the first time
- The compiler will be kept running
- `useSimulatorInstance` will create a dedicated simulator and stop it at the end of the callback